src/datepicker/calendar-header.js

```diff
@@ -405,7 +407,7 @@ export default class CalendarHeader extends React.Component<
         >
           {monthYearTitle}
           <MonthYearSelectIconContainer {...monthYearSelectIconContainerProps}>
-            <TriangleDown />
+            <TriangleDown title="" />
```

Comment:
shouldn't we use `aria-hidden` instead?

Reply:
and `role="presentation"`

Reply:
yeah, I can add the 'presentation' role but the title still seems not useful here